Remove failure from dev-dependency, which in turn dev-depends on failure-derive
and creates circular dependency.
--- a/Cargo.toml	1970-01-01 08:00:00.000000000 +0800
+++ b/Cargo.toml	2019-05-10 11:39:28.396206510 +0800
@@ -34,8 +34,6 @@
 
 [dependencies.synstructure]
 version = "0.10.0"
-[dev-dependencies.failure]
-version = "0.1.0"
 
 [features]
 std = []
